src/main/java/seedu/address/commons/util/StringUtil.java

```diff
@@ -70,6 +70,23 @@ public static boolean isNonZeroUnsignedInteger(String s) {
         }
     }
 
+    /**
+     * Returns Integer if {@code s} represents an unsigned integer
+     * e.g. 0, 1, 2, 3, ..., {@code Integer.MAX_VALUE} <br>
+     * Will return null for any other non-null string input
+     * e.g. empty string, "-1", "+1", and " 2 " (untrimmed), "3 0" (contains whitespace), "1 a" (contains letters)
+     * @throws NullPointerException if {@code s} is null.
+     */
+    public static Integer getInt(String s) {
+        requireNonNull(s);
+
+        try {
+            return Integer.parseInt(s); // "+1" is successfully parsed by Integer#parseInt(String)
+        } catch (NumberFormatException nfe) {
+            return null;
+        }
+    }
+
     public static String clean(String s) {
         return s.strip().replaceAll("[\n|\r|\"|\f|\b|']", "").replaceAll("\\s+", " ").strip();
     }
```

Comment:
It will be better to set `\\s+` as a variable as it is repeated a few times. (Take a look at `containsWordIgnoreCase` method above).